src/main/java/com/conveyal/osmlib/VanillaExtract.java

```diff
@@ -135,7 +138,6 @@ public void service(Request request, Response response) throws Exception {
                 response.setContentType("text/plain");
                 response.setStatus(HttpStatus.BAD_REQUEST_400);
                 outStream.write("URI format: /min_lat,min_lon,max_lat,max_lon[.pbf|.vex] (all coords in decimal degrees)\n".getBytes());
-                ex.printStackTrace();
```

Comment:
Do we not want to print the exception at all? I think we should maybe have separate catch clauses. One for generic exception that prints the exception and one for the IllegalArgumentException, that just prints the error message with URI above.